logstash-core/lib/logstash/pipeline.rb

```diff
@@ -94,60 +112,164 @@ def filters?
   end
 
   def run
+    LogStash::Util.set_thread_name("[#{pipeline_id}]-pipeline-manager")
     @logger.terminal(LogStash::Util::DefaultsPrinter.print(@settings))
 
+    start_workers
+
+    @logger.info("Pipeline started")
+    @logger.terminal("Logstash startup completed")
+
+    @logger.info("Will run till input threads stopped")
```

Comment:
not sure this is useful, any special expectation you're trying to set here for the user?